src/days-2-4-adv/data.py

```diff
@@ -0,0 +1,45 @@
+# ROOMS
```

Comment:
Great Job with project! I really like the effort and creativity put into this. My only concern is readability. Maybe having more descriptive variable names might help others more quickly understand the code?